scripts/gen-nav.js

```diff
@@ -24,7 +24,7 @@ const links = files.map((file) => {
 
 // Case-insensitive sort based on titles (so 'token/ERC20' gets sorted as 'erc20')
 const sortedLinks = links.sort(function (a, b) {
-  return a.title.toLowerCase().localeCompare(b.title.toLowerCase());
+  return a.title.toLowerCase().localeCompare(b.title.toLowerCase(), undefined, { numeric: true });
```

Comment:
What's the consequence of this?

Reply:
Ah sorry I forgot to mention this. That will use [natural sorting](https://en.wikipedia.org/wiki/Natural_sort_order), which will cause ERC1155 to go _after_ ERC20, ERC721 and ERC777 on the API docs, instead of before if using the lexicographic ordering (which looked rather odd).